apps/chat-e2e/src/tests/messageTemplate.test.ts

```diff
@@ -846,7 +869,436 @@ dialTest(
+          'visible',
+        );
+        await variableModalAssertion.assertPromptDescription(
+          `${firstPromptContent(aVarPlaceholder, bVarPlaceholder)} AND ${secondPromptContent(cVarPlaceholder, dVarPlaceholder)}`,
+        );
+        for (const variable of [
+          aVarPlaceholder,
+          bVarPlaceholder,
+          cVarPlaceholder,
+          dVarPlaceholder,
+        ]) {
+          await variableModalAssertion.assertElementAttribute(
+            variableModalDialog.descriptionVar(variable),
+            Attributes.class,
+            ThemeColorAttributes.textAccentTertiary,
+          );
+        }
+      },
+    );
+  },
+);
+
+dialTest(
+  'Replay with Message template: by the owner.\n' +
+    'Replay with Message template: export-import.\n' +
+    'Replay with Message template: duplicated chat',
+  async ({
+    dialHomePage,
+    conversations,
+    conversationDropdownMenu,
+    chatBar,
+    confirmationDialog,
+    conversationAssertion,
+    chat,
+    conversationData,
+    dataInjector,
+    variableModalDialog,
+    variableModalAssertion,
+    apiAssertion,
+    setTestIds,
+  }) => {
+    setTestIds('EPMRTC-4277', 'EPMRTC-4283', 'EPMRTC-4287');
+    let simpleConversationMessage: Conversation;
+    let templateConversationMessage: Conversation;
+    let conversation: Conversation;
+    const firstUpdatedValue = GeneratorUtil.randomString(5);
+    const secondUpdatedValue = GeneratorUtil.randomString(5);
+    let replayName: string;
+
+    await dialTest.step(
+      'Prepare conversation with 2 requests where one of them is based on message template',
+      async () => {
+        simpleConversationMessage =
+          conversationData.prepareDefaultConversation();
+        conversationData.resetData();
+        templateConversationMessage =
+          conversationData.prepareConversationBasedOnMessageTemplate(
+            requestContent,
+            rowsMap,
+          );
+        conversation = conversationData.prepareHistoryConversation(
+          simpleConversationMessage,
+          templateConversationMessage,
+        );
+        await dataInjector.createConversations([conversation]);
+        replayName = `${ExpectedConstants.replayConversation}${conversation.name}`;
+      },
+    );
+
+    await dialTest.step(
+      'Export and then import created conversation',
+      async () => {
+        await dialHomePage.openHomePage();
+        await dialHomePage.waitForPageLoaded();
+        await conversations.openEntityDropdownMenu(conversation.name);
+        await conversationDropdownMenu.selectMenuOption(MenuOptions.export);
+        const exportedData = await dialHomePage.downloadData(() =>
+          conversationDropdownMenu.selectMenuOption(
+            MenuOptions.withoutAttachments,
+          ),
+        );
+        await chatBar.deleteAllEntities();
+        await confirmationDialog.confirm({ triggeredHttpMethod: 'DELETE' });
+        await dialHomePage.importFile(exportedData, () =>
```

Comment:
don't forget to close the toast

Reply:
we have never closed the toast after import, does that make sense?

Reply:
fixed